src/backend/direct.rs

```diff
@@ -1313,7 +1313,7 @@ impl crate::Context for Context {
             sub_range.start,
             wgt::BufferSize::new(size)
         )) {
-            Ok(ptr) => BufferMappedRange {
+            Ok((ptr, size)) => BufferMappedRange {
```

Comment:
Is this right? I took a stab at fixing this compile error.

Reply:
@kvark's PR included the same change: https://github.com/gfx-rs/wgpu-rs/pull/796/files#diff-8d6e6a150805924081efd1ebe6a37810a66a9a1f3b712d0523429f1d65610bceR1316